Agents/PubChemAgent/Dockerfile

```
@@ -1,12 +1,20 @@
 # Install PubChem agent in Docker container
 #==================================================================================================
 # Reference published Docker image for Stack-Client resources to use
-FROM docker.cmclinnovations.com/stack-client:1.6.2 as stackclients
+FROM ghcr.io/theworldavatar/stack-client${IMAGE_SUFFIX}:1.53.0 as stackclients
 
 #------------------------------------------------------
 # Base image to be reused
 #------------------------------------------------------
 FROM python:3.9-slim-buster as base
```

This is quite outdated now and causes a number of problems. I would recommend:
```suggestion
FROM python:3.13-slim as base
```